notebooks/data_center/VM Data Analytics.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
I remember this file already exists with a lower case name. Check it.

Reply:
Yes, I found that there are two folders datacenter and vm_scheduling.